docs/docs/components/actions/README.md

```diff
@@ -0,0 +1,53 @@
+# Actions
```

Comment:
Completely new doc that replaces the doc for legacy actions. Let me know if you think we should add anything else here.